src/features/game/events/landExpansion/plant.ts

```diff
@@ -178,6 +183,25 @@ export function getCropYieldAmount({
     amount *= 1.2;
   }
 
```

Comment:
Looks like we're not checking for `Green Thumb` skills here as well. Could you add it please?

```
  if (skills["Green Thumb"]) {
    amount *= 1.05;
  }
```

Reply:
Done :)